src/modules/commander/commander_params.c

```diff
@@ -340,12 +341,12 @@ PARAM_DEFINE_INT32(COM_OBL_ACT, 0);
  * The offboard loss failsafe will only be entered after a timeout,
  * set by COM_OF_LOSS_T in seconds.
  *
- * @value 0 Position control
- * @value 1 Altitude control
- * @value 2 Manual
- * @value 3 Return to Land
- * @value 4 Land at current position
- * @value 5 Loiter
+ * @value 0 Position mode
+ * @value 1 Altitude mode
+ * @value 2 Manual/Stabilized
```

Comment:
This needs to be checked. I believe that this should be Manual-Stabilized mode in MC and Stabilized mode in FW (rather than Manual mode in FW, which would not be a sensible mode switch).

Reply:
It actually goes into Manual, even in fixed-wing. Not sure if that's intended though.

Reply:
So Manual for FW and Manual/Stabilized for MC (since that has no pure Manual case). I will change appropriately.

The manual is very hard to fly apparently. Would have thought Stabilized made more sense for FW.